catalogs/catalog-lakehouse-iceberg/src/test/java/org/apache/gravitino/catalog/lakehouse/iceberg/integration/test/TestMultipleJDBCLoad.java

```diff
@@ -63,6 +61,36 @@ public void startup() throws IOException {
     postgreSQLContainer = containerSuite.getPostgreSQLContainer();
   }
 
+  @Test
+  public void testJdbcCatalogCreationWithUrlCredentials() throws Exception {
+    String metalakeName = RandomNameUtils.genRandomName("it_metalake");
+    GravitinoMetalake metalake =
+        client.createMetalake(metalakeName, "comment", Collections.emptyMap());
+
+    Map<String, String> icebergMysqlConf = Maps.newHashMap();
+
+    String jdbcUrl =
+        mySQLContainer.getJdbcUrl(TEST_DB_NAME)
+            + "?user="
+            + mySQLContainer.getUsername()
+            + "&password="
+            + mySQLContainer.getPassword();
```

Comment:
both `getUsername` and `getPassword` will return `""`?

Reply:
will return the user and password to access mysql container